packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -551,8 +550,8 @@ export class DropdownBase extends BaseComponent<IDropdownInternalProps, IDropdow
         onMouseMove={this._onItemMouseMove.bind(this, item)}
         role="option"
         aria-selected={isItemSelected ? 'true' : 'false'}
-        ariaLabel={item.ariaLabel || item.text}
-        title={item.title ? item.title : item.text}
+        ariaLabel={item.ariaLabel}
```

Comment:
I understand the change to the aria-Label, but is the change to the title really necessary? This minimally regresses the scenario where the option text is far too long .. now that there's no title, you have no way to tell what the full option text is.

@natalieethell @ecraig12345

Reply:
Using the title attribute is bad practice for accessibility. IMO, its better to fix the accessibility issue for default use cases and allow consumers of the component to decide when to use a title.

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title#Accessibility_concerns
There's some good links here about the issues with the title attribute

Reply:
Hi @ConorWuertz, thanks for the link and I agree with the point you're making. 

However, even though the reliance on title for tooltip purposes is not great, this change here is potentially (and very plausibly) disruptive to everyone already relying on this. All of a sudden, you can no longer mouse over a clipped text in production apps using Fabric Dropdown where it was previously possible. 

I recommend preserving the current treatment of title until such point @ecraig12345 rewrites Dropdown and ComboBox and we can do something better here (and be in a better position to make breaking changes..)

Reply:
Ok, that's a bummer as we'll still see the double-readout issue if we keep the title, but I'll go ahead and adjust this PR to change only the aria-label functionality.